app/controllers/resources_controller.rb

```diff
@@ -1,19 +1,25 @@
 class ResourcesController < ApplicationController
-
   def index
-    per_page = params[:number_of_items_per_page].presence || 25
-    unpaginated = Resource.where(kind: Resource::PUBLISHED_KINDS) #TODO - #FIXME brittle
-                          .includes(:main_image, :gallery_images, :attachments)
-    filtered = unpaginated.search_by_params(params)
-                          .by_created
-    @resources = filtered.paginate(page: params[:page], per_page: per_page).decorate
-
-    @count_display = if filtered.count == unpaginated.count
-                       unpaginated.count
-                     else
-                       "#{filtered.count}/#{unpaginated.count}"
-                     end
-    @sortable_fields = Resource::PUBLISHED_KINDS
+    if turbo_frame_request?
+      per_page = params[:number_of_items_per_page].presence || 25
+      unfiltered = Resource.where(kind: Resource::PUBLISHED_KINDS) # TODO - #FIXME brittle
```

Comment:
FYI 1: i started changing these to `unfiltered` instead of `unpaginated` to be more clear

FYI 2: `total_entries` can't be used anymore now that we're decorating the paginated collection